### PR TITLE
Fix a bunch of small issues to compliment the document set update

### LINKED ISSUE #133: Difficult to tell that Cursor Visibility "disable" options end in L instead of pipe
On docs.microsoft.com, the font in use makes it difficult to tell that the Cursor Visibility sequences that disable blink/show end in a lowercase L rather than a pipe (|).  Only after searching elsewhere did I discover the difference on stackoverflow, where the example was using a serif'd "code" font.

I'm not sure if its possible, but it might be worth displaying all of the Sequence column values in a code block, or whatever fixed with serif'd font is available (I suppose you could even use Cascadia Code ;) ).

### LINKED ISSUE #137: Paragraph Order or Term Definition Misplaced
In the bullet points in the 2nd paragraph, the term GUI is used in the first bullet but defined in the second bullet.  The term should be defined the first time it is used.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a0577daa-0a60-0a8a-e156-f04480e69ba8
* Version Independent ID: 09fe6e63-1060-7661-6891-d909a86b8ba8
* Content: [Creation of a Console - Windows Console](https://docs.microsoft.com/en-us/windows/console/creation-of-a-console)
* Content Source: [docs/creation-of-a-console.md](https://github.com/Microsoft/Console-Docs/blob/master/docs/creation-of-a-console.md)
* Product: **console**
* GitHub Login: @bitcrazed
* Microsoft Alias: **richturn**

### LINKED ISSUE #147: Add disclaimers to APIs that will behave differently in conpty mode
Case in point, `GetConsoleWindow`. We should add a note that this returns a dummy value in conpty mode. We should additionally note that if developers want to make sure their apps work on the broadest category of terminal applications, they should avoid using it. 

See also microsoft/terminal#6887

### LINKED ISSUE #148: Standard Output Handle for this console
It is worth noting that if a process does not have a console before calling this, then function `GetStdHandle(STD_OUTPUT_HANDLE)` will not have returned a valid handle that will then write to this console.

This is often done during startup by code environments - e.g. the Console stream in Dot Net.

To gurantee a valid handle to write to this console you have to call `GetStdHandle(STD_OUTPUT_HANDLE)` _after_ calling it. (And then use the returned handle.)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 54d59483-7cf9-300f-de81-d316509279c2
* Version Independent ID: f0a47b91-e864-8832-5ba3-5fb6400ba8b7
* Content: [AttachConsole function - Windows Console](https://docs.microsoft.com/en-us/windows/console/attachconsole)
* Content Source: [docs/attachconsole.md](https://github.com/Microsoft/Console-Docs/blob/master/docs/attachconsole.md)
* Product: **console**
* GitHub Login: @bitcrazed
* Microsoft Alias: **richturn**

### LINKED ISSUE #158: CONSOLE_READCONSOLE_CONTROL::dwCtrlWakeupMask is a bit mask, not a character
**dwCtrlWakeupMask** is documented to be a control character but it instead seems to be a bit mask in which each bit corresponds to one of the 32 control characters 0x00 to 0x1F.

> ```C
>   ULONG dwCtrlWakeupMask;
> ```

> **dwCtrlWakeupMask**  
> A user-defined control character used to signal that the read is complete.

https://github.com/microsoft/terminal/blob/6c869ebb269155194e668b768d3424eb2a22a125/src/host/readDataCooked.cpp#L495

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 33ea95a4-7dad-cc82-7f3d-3bbc9a75fb33
* Version Independent ID: 4e8ad957-9707-9f59-1a42-e926dcd699b0
* Content: [CONSOLE_READCONSOLE_CONTROL structure - Windows Console](https://docs.microsoft.com/en-us/windows/console/console-readconsole-control)
* Content Source: [docs/console-readconsole-control.md](https://github.com/Microsoft/Console-Docs/blob/master/docs/console-readconsole-control.md)
* Product: **console**
* GitHub Login: @miniksa
* Microsoft Alias: **miniksa**

### LINKED ISSUE #163: Descriptions of CNL and CPL behaviors don't match actual behaviors
The descriptions of the behaviors of CNL/CPL say: "cursor down/up to beginning of nth line in the viewport"

The actual behavior is that n is an offset from the current line.  That could be made clearer.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fc536527-f75d-5a03-7b5a-338a08e40d2f
* Version Independent ID: 097f43d9-243f-466e-1f99-6cfa1f4bfb1f
* Content: [Console Virtual Terminal Sequences - Windows Console](https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences)
* Content Source: [docs/console-virtual-terminal-sequences.md](https://github.com/Microsoft/Console-Docs/blob/master/docs/console-virtual-terminal-sequences.md)
* Product: **console**
* GitHub Login: @miniksa
* Microsoft Alias: **miniksa**

### LINKED ISSUE #167: The "Console Modes" page does not appear in the nav pane
The "Console Modes" page does not appear in the nav pane, located on the left side of the Microsoft Docs website. I was looking for this page, and I was lucky to find it via a link in the ["Input and Output Methods" page](https://docs.microsoft.com/en-us/windows/console/input-and-output-methods).

---

#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ed3cc90a-c50d-d27e-fed0-6c99d9b069a2
* Version Independent ID: f5ded08b-d6cb-0b3c-d9c8-eb097798f582
* Content: [Console Modes - Windows Console](https://docs.microsoft.com/en-us/windows/console/console-modes)
* Content Source: [docs/console-modes.md](https://github.com/Microsoft/Console-Docs/blob/master/docs/console-modes.md)
* Product: **console**
* GitHub Login: @miniksa
* Microsoft Alias: **miniksa**

### LINKED ISSUE #169: Missing link to documentation for "doskey " command.
Missing link to documentation for "doskey " command. Has links for other functions/commands, but not for "doskey".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 831a01bc-cc44-8d31-ac65-36993299e89a
* Version Independent ID: 3bd4cf24-3830-ccc9-bd8c-59e42206417f
* Content: [Console Aliases - Windows Console](https://docs.microsoft.com/en-us/windows/console/console-aliases)
* Content Source: [docs/console-aliases.md](https://github.com/Microsoft/Console-Docs/blob/master/docs/console-aliases.md)
* Product: **console**
* GitHub Login: @miniksa
* Microsoft Alias: **miniksa**

### LINKED ISSUE #170: ClosePseudoConsole talks about an undeclared hOutput identifier
The remarks section talks of a `hOutput` identifier without defining it anywhere on the page.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b02229f7-af1d-32c8-7877-c5afafabbff8
* Version Independent ID: 83b9fd10-3f5d-1391-2c90-9e32de3729b8
* Content: [ClosePseudoConsole function - Windows Console](https://docs.microsoft.com/en-us/windows/console/closepseudoconsole)
* Content Source: [docs/closepseudoconsole.md](https://github.com/Microsoft/Console-Docs/blob/master/docs/closepseudoconsole.md)
* Product: **console**
* GitHub Login: @miniksa
* Microsoft Alias: **miniksa**

### LINKED ISSUE #171: Summary about WriteConsoleInput uses wrong access right
The summary incorrectly states that the `GENERIC_READ` access right is required. The correct one is the `GENERIC_WRITE` access right.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a2ed1408-fab8-5417-680f-5cd5779c911c
* Version Independent ID: 0b4c4aa3-12c6-1d96-ec99-bb59565e0fc2
* Content: [Low-Level Console Input Functions - Windows Console](https://docs.microsoft.com/en-us/windows/console/low-level-console-input-functions)
* Content Source: [docs/low-level-console-input-functions.md](https://github.com/Microsoft/Console-Docs/blob/master/docs/low-level-console-input-functions.md)
* Product: **console**
* GitHub Login: @miniksa
* Microsoft Alias: **miniksa**

### PR DESCRIPTION
**Pull request by miniksa**
Closes #171.
Closes #170. 
Closes #169.
Closes #167.
Closes #163.
Closes #158.
Closes #148.
Closes #147.
Closes #137.
Closes #133.

### CODE REVIEW

**Comment by DHowett** [comment]
GitHub cannot handle `closes x y z a b c`; it needs the verb repeated for every issue. `closes x. closes y. closes z`.

**Comment by miniksa** [comment]
> GitHub cannot handle `closes x y z a b c`; it needs the verb repeated for every issue. `closes x. closes y. closes z`.

GitHub should feel bad about itself. Fixing.

File: docs/attachconsole.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ A process can be attached to at most one console. If the calling process is alre
 
 A process can use the [**FreeConsole**](freeconsole.md) function to detach itself from its console. If other processes share the console, the console is not destroyed, but the process that called **FreeConsole** cannot refer to it. A console is closed when the last process attached to it terminates or calls **FreeConsole**. After a process calls **FreeConsole**, it can call the [**AllocConsole**](allocconsole.md) function to create a new console or **AttachConsole** to attach to another console.
 
+This function is primarily useful to applications that were linked with [**/SUBSYSTEM:WINDOWS**](https://docs.microsoft.com/cpp/build/reference/subsystem-specify-subsystem) which implies to the operating system that a console is not needed before entering the program's main method. In that instance, the standard handles retrieved with [**GetStdHandle**](getstdhandle.md) will likely be invalid on startup until **AttachConsole** is called. The exception to this is if the application is launched with handle inheritance by its parent process.
```
Comment (DHowett) [inline docs/attachconsole.md:67]:
```suggestion
This function is primarily useful to applications that were linked with [**/SUBSYSTEM:WINDOWS**](https://docs.microsoft.com/cpp/build/reference/subsystem-specify-subsystem), which implies to the operating system that a console is not needed before entering the program's main method. In that instance, the standard handles retrieved with [**GetStdHandle**](getstdhandle.md) will likely be invalid on startup until **AttachConsole** is called. The exception to this is if the application is launched with handle inheritance by its parent process.
```

added a comma before `which`

File: docs/closepseudoconsole.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ A handle to an active pseudoconsole as opened by [CreatePseudoConsole](createpse
 
 Upon closing a pseudoconsole, client applications attached to the session will be terminated as well.
 
-A final painted frame may arrive on `hOutput` from the pseudoconsole when this API is called. It is expected that the caller will drain this information from the communication channel buffer and either present it or discard it. Failure to drain the buffer may cause the Close call to wait indefinitely until it is drained or the communication channels are broken another way.
+A final painted frame may arrive on the `hOutput` handle originally received from [CreatePsuedoConsole](createpseudoconsole.md) when this API is called. It is expected that the caller will drain this information from the communication channel buffer and either present it or discard it. Failure to drain the buffer may cause the Close call to wait indefinitely until it is drained or the communication channels are broken another way.
```
Comment (DHowett) [inline docs/closepseudoconsole.md:46]:
```suggestion
A final painted frame may arrive on the `hOutput` handle originally provided to [CreatePsuedoConsole](createpseudoconsole.md) when this API is called. It is expected that the caller will drain this information from the communication channel buffer and either present it or discard it. Failure to drain the buffer may cause the Close call to wait indefinitely until it is drained or the communication channels are broken another way.
```

`houtput` is given, not received.

Comment (miniksa) [reply docs/closepseudoconsole.md:46]: woop, good catch.

File: docs/console-virtual-terminal-sequences.md [hunk 3]
```diff
@@ -101,29 +101,30 @@ Cursor movement will be bounded by the current viewport into the buffer. Scrolli
...
-| ESC \[ ? 25 h | DECTCEM | Text Cursor Enable Mode Show | Show the cursor           |
-| ESC \[ ? 25 l | DECTCEM | Text Cursor Enable Mode Hide | Hide the cursor           |
-
+| ESC \[ ? 12 h | ATT160 | Text Cursor Enable Blinking | Start the cursor blinking |
+| ESC \[ ? 12 l | ATT160 | Text Cursor Disable Blinking | Stop blinking the cursor |
+| ESC \[ ? 25 h | DECTCEM | Text Cursor Enable Mode Show | Show the cursor |
+| ESC \[ ? 25 l | DECTCEM | Text Cursor Enable Mode Hide | Hide the cursor |
 
+> [!TIP]
+> The enable sequences end in a lowercase H character (`h`) and the disable sequences end in a lowercase L character (`l`).
```
Comment (DHowett) [inline docs/console-virtual-terminal-sequences.md:112]: You could even joke about how they look like switch toggles/GPIO lines, [H]igh and [L]ow.

File: docs/getconsolewindow.md [hunk 4]
```diff
@@ -63,6 +63,8 @@ To compile an application that uses this function, define **\_WIN32\_WINNT** as
 
 [!INCLUDE [no-vt-equiv-local-context](./includes/no-vt-equiv-local-context.md)]
 
+For an application that is hosted inside a [**pseudoconsole**](pseudoconsoles.md) session, this function returns a window handle for message queue purposes only. The associated window is not displayed locally as the _pseudoconsole_ is serializing all actions to a stream for presentation on another terminal window elsewhere.
```
Comment (DHowett) [inline docs/getconsolewindow.md:66]: hmm we could mention that it's `unique`. msys2 used it in this way... to identify a single console session

File: docs/getstdhandle.md [hunk 5]
```diff
@@ -70,6 +70,12 @@ Handles returned by **GetStdHandle** can be used by applications that need to re
...
+
+Some applications operate in a mixture of behaviors, for instance a **/SUBSYSTEM:WINDOWS** application might check/use standard handles for logging or debugging purposes but operate normally with a graphical user interface window. These applications will need to carefully probe the state of standard handles on startup and likely make use of [**AttachConsole**](attachconsole.md), [**AllocConsole**](allocconsole.md), and [**FreeConsole**](freeconsole.md) to add/remove a console if desired.
```
Comment (DHowett) [inline docs/getstdhandle.md:75]:
```suggestion
Some applications operate outside the boundaries of their declared subsystem; for instance, a **/SUBSYSTEM:WINDOWS** application might check/use standard handles for logging or debugging purposes but operate normally with a graphical user interface. These applications will need to carefully probe the state of standard handles on startup and make use of [**AttachConsole**](attachconsole.md), [**AllocConsole**](allocconsole.md), and [**FreeConsole**](freeconsole.md) to add/remove a console if desired.
```

**Review by mattwojo — APPROVED** [review]

File: docs/getconsolewindow.md (hunk 4 above)
Comment (miniksa) [reply docs/getconsolewindow.md:66]: Do we mention that? Window handles are unique already by their nature. Do we need to clarify it?

File: docs/console-virtual-terminal-sequences.md (hunk 3 above)
Comment (miniksa) [reply docs/console-virtual-terminal-sequences.md:112]: I, uh.... I'm not sure about that.

File: docs/getconsolewindow.md (hunk 4 above)
Comment (miniksa) [reply docs/getconsolewindow.md:66]: I'm going to just leave it for now.

**Pull request merged**